src/games/ecards.py

```diff
@@ -0,0 +1,275 @@
+import asyncio
+from abstracts import EGameFactory
+
+import random
+
+from interactive import *
+
+from utils import dmerge
+
+
+class ECards(EGameFactory):
+    """
+    EGameFactory for a game where one player is given a prompt,
+    the other players select an answer from a set of safety answers,
+    and the head player selects their favourite answer.
+    """
+
+    # Configuration for the game
+    game_name = "E Cards"
+    game_description = "Prompt cards with your friends!"
+    wait_duration = 6
+    min_players = 2
+    cog_help = """
+    EGameFactory for a game where one player is given a prompt,
+    the other players select an answer from a set of safety answers,
+    and the head player selects their favourite answer.
+    """
```

Comment:
This should ideally be a description of the game's methods -- i.e. what the game is, how to start it, etc.. In the future, the dispatcher will combine all of these variables into a help text for the game.